src/adaptermanager.js

```diff
@@ -218,14 +226,24 @@ exports.aliasBidAdapter = function (bidderCode, alias) {
 
   if (typeof existingAlias === 'undefined') {
     var bidAdaptor = _bidderRegistry[bidderCode];
-
     if (typeof bidAdaptor === 'undefined') {
       utils.logError('bidderCode "' + bidderCode + '" is not an existing bidder.', 'adaptermanager.aliasBidAdapter');
     } else {
       try {
-        let newAdapter = new bidAdaptor.constructor();
-        newAdapter.setBidderCode(alias);
-        this.registerBidAdapter(newAdapter, alias);
+        let newAdapter;
+        let supportedMediaTypes = getSupportedMediaTypes(bidderCode);
+        // Have kept old code to support backward compatibilitiy.
+        // Remove this if loop when all adapters are supporting bidderFactory. i.e When Prebid.js is 1.0
+        if (bidAdaptor.constructor.prototype != Object.prototype) {
+          newAdapter = new bidAdaptor.constructor();
+          newAdapter.setBidderCode(alias);
+        } else {
+          let spec = Object.assign({}, bidAdaptor, { code: alias });
+          newAdapter = newBidder(spec);
```

Comment:
This is a little buggy.

The `bidderFactory` accepts a `spec`, but only returns an object like:

```
{
  callBids: function() { ... }
}
```

So in this code, you're using objects of the shape:

```
newAdapter = newBidder({
  code: alias,
  callBids: function() { ... }
});
```

...but `newBIdder` needs a _spec_. The `typedef` _doesn't_ have `callBids`, but _does_ require several other things which you're not giving here (`interpretResponse`, `buildRequests`, etc).

So... if `callBids` gets called on an _aliased_ bidder, there'll be all sorts of errors about missing functions.